uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-notification.component.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-env amd */
+
+define([
+  'leave-absences/shared/modules/components'
+], function (components) {
+  components.component('leaveNotification', {
+    bindings: {
+      eventName: '<'
+    },
+    templateUrl: ['shared-settings', function (sharedSettings) {
+      return sharedSettings.sharedPathTpl + 'components/leave-notification.html';
+    }],
+    controllerAs: 'notification',
+    controller: ['$log', '$rootScope', 'pubSub', 'LeaveRequest', controller]
+  });
+
+  function controller ($log, $rootScope, pubSub, LeaveRequest) {
+    $log.debug('Component: leave-notification');
+
+    var filters = {};
+    var vm = this;
+    vm.count = 0;
+    vm.loading = { count: true };
```

Comment:
This is something we have followed in all components lately, to keep this structure. This way we have a consistent code, also in case we need to add another loader we dont need to change the structure of the variable.